### PR TITLE
🍱 Dashboard adaption for Nextcloud 20

### PR DESCRIPTION
**Pull request by juliushaertl**
WIP, this might need some more work based on the outcome of the design discussions, however this already shows how a panel can be registered.

### CODE REVIEW

**Comment by jancborchardt** [comment]
Works well :+1: added some layout fixes of extension and removed the subline when entries are shown in the Dashboard for now.

File: lib/AppInfo/LoadAdditionalScriptListener.php [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+namespace OCA\Recommendations\AppInfo;
```
Comment (ChristophWurst) [inline lib/AppInfo/LoadAdditionalScriptListener.php:24]: I'd prefer `OCA\Recommendations\Listeners`

Comment (juliushaertl) [reply lib/AppInfo/LoadAdditionalScriptListener.php]: Fixed in #244 already

**Comment by jancborchardt** [comment]
Needs a rebase @juliushaertl

**Pull request merged**